SPECS/kubernetes/kubernetes-1.17.11.signatures.json

```diff
@@ -1,5 +1,6 @@
 {
  "Signatures": {
+  "golang-1.15-k8s-1.17-test.patch": "4607a1675331e309f651737cd10114e2a610dcd0250ec06a46e4de7feb000161",
```

Comment:
I think we don't add signatures for patches. Not entirely sure why.

Reply:
this is not a normal .patch file because kubernetes sources are contained in a tarball within the src tarball we get from ms-kubernetes's team... you have to untar the tarball in %setup and then untar the 'real' src tarball and then apply the patch. This forced me to get this patch file from source2 in spec (instead of patch4)